app/src/types/ATSClientResource.ts

```diff
@@ -0,0 +1,23 @@
+import { BasicResponse } from '../utils/enums/application';
+
+type AddressResource = {
+  '@type': 'AddressResource';
```

Comment:
Cannot assign a default value in a type. This is assigning `@type` as a type of `AddressResource`

Reply:
sorry, I missed this. done